.github/workflows/promptflow-evals-unit-test.yml

```diff
@@ -0,0 +1,99 @@
+name: promptflow-evals-unit-test
+
+on:
+  schedule:
```

Comment:
Is this needed to run on schedule ?

Reply:
Maybe this is for some sort of daily scheduled report? @wangchao1230, @crazygao?

Reply:
Yes. This can be used as data source for daily scheduled report.